pkg/controllers/clusterresourceplacement/controller.go

```diff
@@ -20,6 +22,17 @@ import (
 	"go.goms.io/fleet/pkg/utils/controller"
 )
 
+const (
+	// ApplyFailedReason is the reason string of placement condition when the selected resources fail to apply.
+	ApplyFailedReason = "ApplyFailed"
```

Comment:
Reason should be a camelCase string.

Reply:
Same for the other reasons down below.

Reply:
i checked the kubernetes repo. They are using "CamelCase".

eg, https://github.com/openshift/kubernetes/blob/af29f64acbf9e6d33cde7f3194cf858f54058d3b/staging/src/k8s.io/api/extensions/v1beta1/types.go#L1237

another example, https://github.com/kubernetes/kubernetes/blob/e3bc35bc1bc7b399b564f2c4efc75eb9959e70f7/pkg/kubelet/kubelet_node_status.go#L341